### PR TITLE
fix(vara): Add staking admin origin to pallet-staking

### PR DESCRIPTION
**Pull request by ukint-vs**
Staking Admin should be able to manage staking pallet as well as the root

@reviewer-or-team

### CODE REVIEW

**Review by breathx — APPROVED** [review]

File: runtime/vara/src/lib.rs [hunk 1]
```diff
@@ -568,7 +568,7 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
     type Fallback = onchain::OnChainExecution<OnChainSeqPhragmen>;
     type GovernanceFallback = onchain::OnChainExecution<OnChainSeqPhragmen>;
     type Solver = SequentialPhragmen<AccountId, SolutionAccuracyOf<Self>, ()>;
-    type ForceOrigin = EnsureRoot<AccountId>;
+    type ForceOrigin = AdminOrigin;
```
Comment (grishasobol) [inline runtime/vara/src/lib.rs:571]: Could you explain please why do we need stacking admin in elections?

Comment (ukint-vs) [reply runtime/vara/src/lib.rs:571]: To control this pallet. Like emergency solution result, min untrusted score, governance fallback

Comment (grishasobol) [reply runtime/vara/src/lib.rs:571]: As I can see in doc StakingAdmin is only for slashes
https://github.com/gear-tech/gear/blob/e982f460bc17b0fdebd382f899ed80d0c353825c/runtime/vara/src/governance/origins.rs#L37C1-L39

Comment (ukint-vs) [reply runtime/vara/src/lib.rs:571]: updated

**Review by grishasobol — APPROVED** [review]

**Pull request merged**